README.md

```diff
@@ -4,10 +4,11 @@
 
 The following parameters have been prefixed with the `siren` keyword in order to avoid conflict with users possibly setting Netty's original parameters:
 
-- io.netty.recycler.maxCapacityPerThread;
-- io.netty.recycler.maxCapacity;
-- io.netty.buffer.bytebuf.checkAccessible; and
-- io.netty.noUnsafe.
+- io.netty.recycler.maxCapacityPerThread
+- io.netty.maxDirectMemory
+- io.netty.recycler.maxCapacity
+- io.netty.buffer.bytebuf.checkAccessible
+- io.netty.noUnsafe
```

Comment:
better to use somethign more generic, e.g., all the netty's parameter `io.netty.*` were prefixed with the `siren` keyword ...